whats_on/app/views/components/exhibits/exhibits.njk

```diff
@@ -0,0 +1,30 @@
+
+<div>
```

Comment:
Could this one be a `Fragment` (assuming it's only there to stop React complaining)?

Reply:
This is actually there for the async component, we're still in nunjucks and here.